content/Watsonx/WatsonxAI/files/rest-api.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Accesing Watsonx.ai via REST API\n",
+    "\n",
+    "In this lab, we will take a comprehensive look at making HTTP requests to access [Watsonx.ai's REST API](https://bam.res.ibm.com/docs/api-reference) and learn how to use the functionality. This lab explore only a few of the many REST endpoints available so explore the REST API documentation to view the full list of capabilities.\n",
```

Comment:
The bam.res.ibm.com will be internal to IBMers.  Hopefully we have a legit API doc link for GA version before we start doing workshops.  Otherwise we'll need to remove this link.  I'll try to track down the API doc link for GA.